pkg/apis/operator/v1alpha1/common.go

```diff
@@ -62,6 +62,15 @@ type KComponentStatus interface {
 	SetVersion(version string)
 }
 
+// HighAvailability specifies options for deploying Knative Serving control
+// plane in a highly available manner. Note that HighAvailability is still in
+// progress and does not currently provide a completely HA control plane.
+type HighAvailability struct {
```

Comment:
nit: Please put this below the CommonSpec, in the same order as the fields are specified on CommonSpec.